new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiDomain.java

```diff
@@ -504,12 +534,204 @@ public void testCreateMiiDomainSameDomainUidDiffNS() {
     }
   }
 
+  @Test
+  @Order(4)
+  @DisplayName("Update the sample-app application to version 2")
+  @Slow
+  @MustNotRunInParallel
+  public void testPatchAppV2() {
+    
+    // app in the new image contains what is in the original app dir sample-app, 
+    // plus the replacements or/and additions in the second app dir sample-app-2.
+    final String appDir1 = "sample-app";
+    final String appDir2 = "sample-app-2";
+    final String adminServerPodName = domainUid + "-admin-server";
+    final String managedServerPrefix = domainUid + "-managed-server";
+    final int replicaCount = 2;
+    
+    // The verification of app's continuous availability during patching is turned off
```

Comment:
My understanding is that the application should stay up when the domain is patched because the restart of the next pod will not begin before the current pod completes its restart. But I guess there is no way we can guarantee continuous availability. 

Modified the comment.